### ISSUE TITLE
4.3.0 release plan

### ISSUE THREAD

**Issue by tk0miya** [sphinx-doc/sphinx#9824]
Hi maintainers,

About 2months has passed since the last stable release. Based on our release cycle, I'll ship a new package 4.3.0 in nearly days. Do you have any concerns?

* Initially, the release was planned in this weekend. But I did not have time to work for the release and could not post this release plan this week. So I'd like to postpone it for a few days.

Note: issues marked as 4.3.0:
https://github.com/sphinx-doc/sphinx/milestone/114

**Comment by jmckenna** [comment]
sounds good, thanks @tk0miya-san

**Comment by jakobandersen** [comment]
No concerns from my side.

**Comment by shimizukawa** [comment]
LGTM!

**Comment by jakobandersen** [comment]
> No concerns from my side.

Well, that didn't last long: a variation of #9833 should probably go into 4.x before release.

**Comment by tk0miya** [comment]
I will ship a new version after #9833 will be merged. Maybe within 24 hours.

**Comment by tk0miya** [comment]
Just released! Thank you for all your work!

**Comment by jmckenna** [comment]
@tk0miya for your release steps, can you also add: 
- generate a release here on GitHub (this way all of those 'following' this repository are automatically notified of the release)
- for example, see: https://github.com/MapServer/MapServer/releases/tag/rel-7-6-4
- specifically:
  - go [here ](https://github.com/sphinx-doc/sphinx/releases)and click the button "Draft a new release"

thanks!

**Comment by tk0miya** [comment]
@jmckenna Could you explain (or give me a pointer) the advantage of the GitHub releases? TBH, I'd not like to increase tasks on releasing...

**Comment by hugovk** [comment]
One advantage is people can watch repos for releases only, so they'll get a notification when there's a new release:

![image](https://user-images.githubusercontent.com/1324225/141332394-b082b698-1410-4247-9613-7f5487ca6a9b.png)

It doesn't ping for new tags.

I don't have experience myself, but there are some GitHub Actions out there to automatically create a release for tags.

**Comment by tk0miya** [comment]
It would be nice if GHA automatically generates a release on tagging :-)

**Comment by tk0miya** [comment]
Note: https://github.com/actions/create-release

**Comment by tk0miya** [comment]
It's helpful if somebody posts a PR :-)

**Comment by jmckenna** [comment]
@tk0miya if you click that button that I said earlier, a new tag is auto-generated for the release, and the archives are auto-generated for the release, all you have to do is paste the text: "Sphinx 4.3.0 release is now available, as a bug-fix release.  Please see the changelog at https://www.sphinx-doc.org/en/master/changes.html Thank-you to the whole community."    <--and the 130+ "watchers" for this repository are auto-magically notified.

**Comment by hugovk** [comment]
Please see PR https://github.com/sphinx-doc/sphinx/pull/9842.

https://github.com/actions/create-release is archived, so of the ones linked from there, this uses the one with the most stars: https://github.com/softprops/action-gh-release

**Comment by tk0miya** [comment]
I merged #9842 now. Please wait for the next release.
